app/services/support_interface/send_initial_emails.rb

```diff
@@ -0,0 +1,30 @@
+module SupportInterface
+  class SendInitialEmails
+    attr_accessor :ucas_match
+
+    def initialize(ucas_match:)
+      self.ucas_match = ucas_match
+    end
+
+    def call
+      return false if ucas_match.initial_emails_sent?
+
+      return ::SendInitialEmailsMultipleAcceptances.new(ucas_match).call if ucas_match.application_accepted_on_ucas_and_accepted_on_apply?
```

Comment:
`SendInitialEmailsMultipleAcceptances` is not particularly complex, I'd say it's on a par with the `CandidateMailer` and `ProviderMailer` code below. Is there a reason we have `SendInitialEmailsMultipleAcceptances` as a service, while the code below isn't one?

Reply:
I started with `SendInitialEmails` but when I added the multiple acceptances part it became a bit too noisy and I wanted to seperate the code, but maintain the abstraction as the controller shouldn't care what type of initial email it is. But I agree, moving the duplicate acceptances code to it's own service makes sense and probably makes things a bit easier to understand.